src/App.tsx

```diff
@@ -1,27 +1,46 @@
 import React from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { Good } from './types/Good';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRed } from './api/goods';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+export const App: React.FC = () => {
+  const [goodsList, setGoodsList] = React.useState<Good[]>([]);
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+  const goodsSetter = (callback: Good[]) => {
```

Comment:
should there be should a callback? Now it's an array of goods. But maybe you really can receive callback and all it to load the needed array with goods
```suggestion
  const goodsSetter = (callback: Good[]) => {
```